doc/users/next_whats_new/asinh_scale.rst

```diff
@@ -0,0 +1,9 @@
+New axis scale ``asinh``
+------------------------
+
+The new ``asinh`` axis scale offers an alternative to ``symlog`` that
```

Comment:
It would be nice to include a plot for the what's new (there should be examples of that, but basically search for the `.. plot:` directive), and link to the new demo...